CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Major: Added support for Twitch's Chat Replies. [Wiki Page](https://wiki.chatterino.com/Features/#message-replies) (#3722)
 - Major: Added multi-channel searching to search dialog via keyboard shortcut. (Ctrl+Shift+F by default) (#3694, #3875)
+- Minor: Load missing messages from Recent Messages API upon reconnecting (#3878)
```

Comment:
Is the changelog entry intentionally at the top of the "Minor" section? I was under the impression we usually place them at the bottom of the respective section

Reply:
I was under the impression we placed new entries at the top of the section 😅 But I went ahead and reordered it